api/components/examples/health-insurance/on_status/on_status_renew_processing.yaml

```diff
@@ -186,13 +155,93 @@ message:
                   code: SETTLEMENT_AMOUNT
                 value: '400'
               - descriptor: 
-                  code: OFFLINE_CONTRACT
```

Comment:
this seems like by mistake

Reply:
This tag is still part of SETTLEMENT_TERMS but shifted to another line.